src/goose/cli/prompt/lexer.py

```diff
@@ -1,5 +1,5 @@
 import re
-from typing import Callable, List, Tuple
+from typing import Callable
```

Comment:
in cases like this, I couldn't use the primitive as it wasn't subscriptable

Reply:
fwiw this `Callable` was deprecated which kind of solves the subscriptable problem. we'd probably always need to keep one of those since i don't recall it's hinting can be expressed otherwise

https://docs.python.org/3/library/typing.html#typing.Callable
> Deprecated since version 3.9: [collections.abc.Callable](https://docs.python.org/3/library/collections.abc.html#collections.abc.Callable) now supports subscripting ([]). See [PEP 585](https://peps.python.org/pep-0585/) and [Generic Alias Type](https://docs.python.org/3/library/stdtypes.html#types-genericalias).

Reply:
me neither.. if you figure it out, ping me!